polars/polars-ops/src/frame/pivot/mod.rs

```diff
@@ -246,7 +246,7 @@ fn pivot_impl(
                 debug_assert_eq!(value_agg_phys.len(), row_locations.len());
 
                 let mut cols = if value_agg_phys.dtype().is_numeric() {
-                    macro_rules! dispatch {
+                    macro_rules! pivot {
```

Comment:
Right. This is purely for error reporting, it will include name of the macro if it fails for whatever reason. 

Btw (1): here's another case when you check `is_numeric()`. Is decimal numeric or not? :) For pivoting purposes - sure! For many other purposes, no... 🤦 

Btw (2), it just occurred to me, as this is yet another point where you check if something is numeric in point 1 but use this implicit assumption in point 2, wouldn't it be cleaner to instead have
```rust
fn as_numeric(&self) -> Option<NumericChunked<'_>> {
    match self.dtype() { ... }
}
enum NumericChunked<'a> {
    UInt8(&'a UInt8Chunked),
    ...
}
```
Yet another case where we can potentially dodge implicit assumptions and unwraps by using type system?